app/models/question.rb

```diff
@@ -1,19 +1,24 @@
 # frozen_string_literal: true
 
-class ApplicationCriteria
+class Question
```

Comment:
Can you help me understand more why it is valuable to have both a question model and also an ApplicationCriteria model?

Reply:
the container is ApplicationCriteria ... It has many questions but Question doesn't necessarily need persistence in Database ... so we have a Poro object that allows us to have properties like required?